src/main/resources/wa-task-configuration-ia-asylum.dmn

```diff
@@ -64,6 +64,26 @@
           <text>if (caseData.caseManagementLocation != null and caseData.caseManagementLocation.baseLocation != null  and caseData.staffLocation != null) then caseData.staffLocation else "Taylor House"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_03afipt">
+        <inputEntry id="UnaryTests_0aqybrx">
+          <text>-</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_04r7w97">
+          <text>"caseManagementCategory"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_099dwbs">
+          <text>if (caseData.caseManagementCategory !=null and caseData.caseManagementCategory.value.code = "refusalOfHumanRights") then "Human rights" 
+else if (caseData.caseManagementCategory !=null and caseData.caseManagementCategory.value.code = "refusalOfEu") then "EEA" 
+else if (caseData.caseManagementCategory !=null and caseData.caseManagementCategory.value.code = "deprivation") then "DoC" 
+else if (caseData.caseManagementCategory !=null and caseData.caseManagementCategory.value.code = "protection") then "Protection" 
+else if (caseData.caseManagementCategory !=null and caseData.caseManagementCategory.value.code = "revocationOfProtection") then "Revocation"
+else if caseData.appealType = "refusalOfHumanRights" then "Human rights" 
+else if caseData.appealType = "refusalOfEu" then "EEA" 
+else if caseData.appealType = "deprivation" then "DoC" 
+else if caseData.appealType = "protection" then "Protection" 
+else if caseData.appealType = "revocationOfProtection" then "Revocation" else ""</text>
```

Comment:
what happens if caseData.caseManagementCategory is null and caseData.appealType is also null

Reply:
At the moment is an empty string but should this be null ? or was this written as part of the ticket as an AC ?

Reply:
i am assuming that a case without an appealType is valid because ia-case-api throws an exception, this is the case for caseManagementCategory for new cases. but old cases may not have the new field, that is why I am checking if the field is null.
the empty string was present in the appealType field, so because the 2 fields should contain similar values, I kept for the new field